### PR TITLE
Feature: add set zoom and coordinates features

### PR DESCRIPTION
**Pull request by Lucifer-02**
Those features are implemented by adding more parameters to the Google Maps URL:
- zoom: Change the zoom level in the map view.
- coordinates: set coordinates to search on the map.

### CODE REVIEW

File: main.go
Comment (gosom) [inline main.go:1]: Do you mind also updating the README with the new options?

**Comment by gosom** [comment]
@Lucifer-02 thanks for the pull request. Appreciated

Do you mind to take care of the linter issues also?

```
  Running [/home/runner/golangci-lint-1.61.0-linux-amd64/golangci-lint run --out-format=github-actions] in [] ...
  Warning: var-naming: var mapUrl should be mapURL (revive)
  Error: if statements should only be cuddled with assignments (wsl)
  
  level=warning msg="[config_reader] The output format `github-actions` is deprecated, please use `colored-line-number`"
  
  Error: issues found
```

**Review by gosom — CHANGES_REQUESTED** [review]

File: main.go
Comment (Lucifer-02) [reply main.go:1]: Sure

**Comment by Lucifer-02** [comment]
Sorry for this carelessness!

File: gmaps/job.go [hunk 1]
```diff
@@ -33,11 +33,19 @@ func NewGmapJob(id, langCode, query string, maxDepth int, extractEmail bool) *Gm
 		id = uuid.New().String()
 	}
 
+	mapURL := fmt.Sprintf("https://www.google.com/maps/search/%s", query)
+	if geoCoordinates != "" {
+		mapURL = fmt.Sprintf("https://www.google.com/maps/search/%s/@%s", query, strings.ReplaceAll(geoCoordinates, " ", ""))
+	}
+	if zoom > 0 {
```
Comment (gosom) [inline gmaps/job.go:40]: linter fails here 

```
mapURL := fmt.Sprintf("https://www.google.com/maps/search/%s", query)

if geoCoordinates != "" {
		mapURL = fmt.Sprintf("https://www.google.com/maps/search/%s/@%s", query, strings.ReplaceAll(geoCoordinates, " ", ""))
	}

if zoom > 0 {
    mapURL = fmt.Sprintf("%s,%dz", mapURL, zoom)
}

```

PS: you can use `make lint` before commiting. 

Thank you

File: gmaps/job.go [hunk 2]
```diff
@@ -33,11 +33,19 @@ func NewGmapJob(id, langCode, query string, maxDepth int, extractEmail bool) *Gm
 		id = uuid.New().String()
 	}
 
+	mapURL := fmt.Sprintf("https://www.google.com/maps/search/%s", query)
+	if geoCoordinates != "" {
+		mapURL = fmt.Sprintf("https://www.google.com/maps/search/%s/@%s", query, strings.ReplaceAll(geoCoordinates, " ", ""))
```
Comment (gosom) [inline gmaps/job.go:38]: this url seems not to do the expected.

but this looks to work:

https://www.google.com/maps/search/restaurant/?ll=37.97945,23.71622&z=20

can you please verify and adjust?

**Review by gosom — CHANGES_REQUESTED** [review]

**Comment by gosom** [comment]
@Lucifer-02 can you also please resolve the conflics with the main?

P.S. I can do my self but I guess you prefer to have the commits. If that is not the case then I can takeover the PR. 

Thanks again for your contribution

**Comment by Lucifer-02** [comment]
> @Lucifer-02 can you also please resolve the conflics with the main?
> 
> P.S. I can do my self but I guess you prefer to have the commits. If that is not the case then I can takeover the PR.
> 
> Thanks again for your contribution

File: gmaps/job.go (hunk 2 above)
Comment (Lucifer-02) [reply gmaps/job.go:38]: The search feature does not work with your URL

**Comment by Lucifer-02** [comment]
> @Lucifer-02 can you also please resolve the conflics with the main?
> 
> P.S. I can do my self but I guess you prefer to have the commits. If that is not the case then I can takeover the PR.
> 
> Thanks again for your contribution

Yeah, this is my first PR and I'm a newbie at Go

File: gmaps/job.go (hunk 1 above)
Comment (Lucifer-02) [reply gmaps/job.go:41]: I see

File: gmaps/job.go (hunk 2 above)
Comment (Lucifer-02) [reply gmaps/job.go:38]: I checked again, and my URL only works with the zoom parameter. But yours does not work. I will adjust the code.

Comment (gosom) [reply gmaps/job.go]: so we will have only the zoom right?

**Review by gosom — APPROVED** [review]

**Comment by gosom** [comment]
@Lucifer-02 merged. 

Thank you .

Nice job!

**Pull request merged**